lib/data_update_scripts/20211104200856_update_each_articles_privileged_user_point_count.rb

```diff
@@ -0,0 +1,13 @@
+module DataUpdateScripts
+  class UpdateEachArticlesPrivilegedUserPointCount
+    def run
+      # This code is "idempotent" in that it's calculating and caching a value stored elsewhere.
+      Article.find_in_batches do |articles|
+        articles.each do |article|
+          article.update_column(:privileged_users_reaction_points_sum,
```

Comment:
There is an optimization available here based on the 0 default column value - you only need to update if `article.reactions.privileged_category.exists?` I think (or if sum is nonzero). 

We're issuing update statements needlessly (update_column here bypasses any normal dirty state tracking on the object) setting to zero where it was already.

Reply:
Good catch and updated accordingly.